a.txt

```diff
@@ -0,0 +1,134 @@
+commit 5b04a8ac3458a9e25c65ba022adbbc41653cf8f7
```

Comment:
remove a.txt file, i guess it was added by mistake

Reply:
ups, correct, this should not have been included..